packages/cli/package.json

```diff
@@ -23,10 +23,12 @@
   },
   "dependencies": {
     "@zowe/zos-uss-for-zowe-sdk": "^8.11.0",
+    "terminal-kit": "^3.1.2",
```

Comment:
While this is a really cool library, there may be a few things we have to consider.
For instance, transitive dependencies that have not being maintained for a long time.
We can discuss offline, but here is a graph that might help when adding new dependencies 😋 
https://npmgraph.js.org/?q=terminal-kit